internal/route_table_context_test.go

```
@@ -0,0 +1,150 @@
+		t.Fatalf("input[route_table] should contain the raw RouteTable map")
+	}
+	if routeTableMap["RouteTableId"] != "rtb-public" {
+		t.Fatalf("route_table.RouteTableId = %v, want rtb-public", routeTableMap["RouteTableId"])
+	}
+
+	contextMap, ok := input["route_table_context"].(map[string]interface{})
+	if !ok {
+		t.Fatalf("input[route_table_context] should be a map")
+	}
+
+	current := contextMap["current"].(map[string]interface{})
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)